app/logic/Calendar/ActiveSync/ActiveSyncEvent.ts

```diff
@@ -120,25 +120,18 @@ export class ActiveSyncEvent extends Event {
   }
 
   async saveToServer(): Promise<void> {
-    let organizer;
-    if (this.participants.length && this.myParticipation == InvitationResponse.Organizer) {
-      organizer = this.participants.find(participant => participant.emailAddress == this.calendar.account.emailAddress);
-      if (organizer) {
-        organizer.response = InvitationResponse.Organizer;
-      } else {
-        organizer = new Participant(this.calendar.account.emailAddress, null, InvitationResponse.Organizer);
-        this.participants.add(organizer);
-      }
-    }
     // Not supporting tasks for now.
     if (this.parentEvent) {
       this.parentEvent.saveFields(this.parentEvent.toFields(this.toFields()));
     } else {
       await this.saveFields(this.toFields());
+      if (!this.calUID) {
+        // If we haven't set a UID yet, Exchange will auto-generate one,
+        // so sync up to find out what it is.
+        await this.calendar.listEvents();
```

Comment:
How do we match this event with the one that's coming from the server, given that we don't have the UID to match it?